src/fiatExchanges/ExchangeQR.tsx

```diff
@@ -0,0 +1,191 @@
+import { RouteProp } from '@react-navigation/native'
```

Comment:
nit: Should this file be named `ExchangeQRScreen.tsx` to stay consistent with other files in this folder? Although I guess we also `ExternalExchanges.tsx` so don't feel too strongly

Reply:
I'm not sure what convention we ought to be using for screen naming; I recall hearing conflicting opinions in the past about whether or not to suffix with `Screen`. I like having a `Screen` suffix for screens, but not sure what other folks think.